### ISSUE TITLE
nvda does not read the list of attachments in outlook 

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#12265]
### Steps to reproduce:
open an email in outlook that has an attachment.
Press shift + tabe to focus on the list of attachments 
### Actual behavior:
nvda does not speak, just an error sound 
### Expected behavior:
nvda should read the name of the attachments 
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-22222,2e37433e

#### Windows version:
10 19043.906
#### Name and version of other software in use when reproducing the issue:
OUTLOOK.EXE, Microsoft Outlook 16.0.13801.20360

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
yes
I noticed the mistakes in the last alphas, I can't speak from which 
#### If add-ons are disabled, is your problem still occurring?
yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
no
log:
[outlook.txt](https://github.com/nvaccess/nvda/files/6251782/outlook.txt)

**Comment by CyrilleB79** [comment]
Reproduced also on my side on 2 computers.

The regression seems to have appeared with #12210.
Cc @michaelDCurran, author of this PR.

**Comment by Nikita34196** [comment]
Hello. I also confirm this. . I think the problem also affects some windows settings. there, too, there was a regression.
